tools/data/resize_videos.py

```diff
@@ -17,6 +17,10 @@ def resize_videos(vid_item):
         bool: Whether generate video cache successfully.
     """
     full_path, vid_path = vid_item
+    if args.to_mp4:
```

Comment:
Need a comment to explain these codes lines

Reply:
Done